core/templates/dev/head/pages/exploration_player/tutor_card_directive.html

```diff
@@ -103,7 +103,7 @@
         <i class="material-icons md-18" style="position: relative; top: 3px;">&#xE5C8;</i>
       </div>
       <div ng-if="isViewportNarrow()">
-        <md-button class="instructions-button" ng-click="onDismiss()">
+        <md-button class="instructions-button"  ng-click="onDismiss()">
```

Comment:
nit: remove extra space between 'class="instructions-button"' and ng-click="onDismiss()" (should be one space instead of two).

Reply:
Hi @tjiang11 , I will correct it . Thanks for reviewing .